modules/security/src/test/java/com/devonfw/module/security/common/impl/accesscontrol/AccessControlConfigSimple.java

```diff
@@ -75,13 +75,13 @@ public AccessControlConfigSimple() {
     super();
     AccessControlGroup readMasterData = group(GROUP_READ_MASTER_DATA, PERMISSION_FIND_OFFER, PERMISSION_FIND_PRODUCT,
         PERMISSION_FIND_STAFF_MEMBER, PERMISSION_FIND_TABLE);
-    AccessControlGroup cook = group(GROUP_COOK, readMasterData, PERMISSION_FIND_ORDER, PERMISSION_SAVE_ORDER,
+    AccessControlGroup user3 = group(GROUP_USER3, readMasterData, PERMISSION_FIND_ORDER, PERMISSION_SAVE_ORDER,
         PERMISSION_FIND_ORDER_POSITION, PERMISSION_SAVE_ORDER_POSITION);
-    AccessControlGroup barkeeper = group(GROUP_BARKEEPER, cook, PERMISSION_FIND_BILL, PERMISSION_SAVE_BILL,
+    AccessControlGroup user2 = group(GROUP_USER2, user3, PERMISSION_FIND_BILL, PERMISSION_SAVE_BILL,
         PERMISSION_DELETE_BILL, PERMISSION_DELETE_ORDER);
-    AccessControlGroup waiter = group(GROUP_WAITER, barkeeper, PERMISSION_SAVE_TABLE);
+    AccessControlGroup user1 = group(GROUP_USER1, user2, PERMISSION_SAVE_TABLE);
     // AccessControlGroup chief =
-    group(GROUP_CHIEF, waiter, PERMISSION_SAVE_OFFER, PERMISSION_SAVE_PRODUCT, PERMISSION_SAVE_STAFF_MEMBER,
+    group(GROUP_ADMIN, user1, PERMISSION_SAVE_OFFER, PERMISSION_SAVE_PRODUCT, PERMISSION_SAVE_STAFF_MEMBER,
         PERMISSION_DELETE_OFFER, PERMISSION_DELETE_PRODUCT, PERMISSION_DELETE_STAFF_MEMBER,
```

Comment:
This is just a test for our module to test `AccessControlConfig`. Your changes are OK, but would not have been needed.